src/mmw/js/src/draw/views.js

```diff
@@ -456,7 +458,9 @@ var WatershedDelineationView = Marionette.ItemView.extend({
                 displayAlert(message, modalModels.AlertTypes.warn);
             })
             .always(function() {
-                map.removeLayer(rwdClickedPoint);
+                if (map && rwdClickedPoint) {
```

Comment:
Same comment as above, `rwdClickedPoint` seems to be the only reference that might be `null`.